test/torchaudio_unittest/functional/batch_consistency_test.py

```diff
@@ -232,3 +232,30 @@ def test_lfilter(self):
         ])
 
         self.assertEqual(batchwise_output, itemwise_output)
+
+    def test_filtfilt(self):
+        num_filters = 2
+        num_order = 3
+        torch.manual_seed(296)
+
+        waveform = common_utils.get_whitenoise(
+            sample_rate=8000, n_channels=self.batch_size * num_filters
+        )
+        waveform = waveform.view(self.batch_size, num_filters, waveform.size(-1))
+        a_coeffs = torch.rand(num_filters, num_order + 1)
+        b_coeffs = torch.rand(num_filters, num_order + 1)
+
+        self.assert_batch_consistency(F.filtfilt, waveform, a_coeffs, b_coeffs)
+
+    def test_filtfilt_filterbank(self):
+        num_filters = 2
+        num_order = 3
+        torch.manual_seed(296)
+
+        waveform = common_utils.get_whitenoise(sample_rate=8000, n_channels=4)
+        a_coeffs = torch.rand(num_filters, num_order + 1)
+        b_coeffs = torch.rand(num_filters, num_order + 1)
+
+        self.assert_batch_consistency(
+            F.filtfilt, waveform, a_coeffs, b_coeffs, batching=False
+        )
```

Comment:
These type of tests might not be neccessary. 
I recoommend only test the batching behavior of filter coefficients, like how `test_lfilter` do.